### PR TITLE
added env reading feature

### PR DESCRIPTION
**Pull request by vijeyash1**

### CODE REVIEW

File: clickhouse/clickhouse_client.go [hunk 1]
```diff
@@ -5,14 +5,20 @@ import (
...
 )
 
+//reading the interserver_http_port from env variable
+//var port string = os.Getenv("interserver_http_port")
+
+var dbAdress string = os.Getenv("DB_ADDRESS")
+
 func GetClickHouseConnection() (*sql.DB, error) {
-	connect, err := sql.Open("clickhouse", "tcp://clickhouse:9009?debug=true")
+	connect, err := sql.Open(dbAdress, "tcp://clickhouse:9009?debug=true")
```
Comment (avinashkna4) [inline clickhouse/clickhouse_client.go:21]: 1. you have modified database type . First parameter should not be changed. Please revert it  
2. read both host and port separately and replace envs in second parameter "tcp://clickhouse:9009?debug=true")

File: agent/k8smetrics_agent.go [hunk 2]
```diff
@@ -40,10 +41,16 @@ const (
...
+//to read the token from env variables
+var token string = os.Getenv("NATS_TOKEN")
+
+var natsurl string = os.Getenv("NATS_ADDRESS")
+
 func main() {
 
 	// Connect to NATS
-	nc, err := nats.Connect("nats", nats.Name("K8s Metrics"), nats.Token("UfmrJOYwYCCsgQvxvcfJ3BdI6c8WBbnD"))
+	//, nats.Token("UfmrJOYwYCCsgQvxvcfJ3BdI6c8WBbnD")
```
Comment (avinashkna4) [inline agent/k8smetrics_agent.go:52]: remove this comment

File: client/k8smetrics_client.go [hunk 3]
```diff
@@ -13,10 +14,17 @@ import (
...
+var token string = os.Getenv("NATS_TOKEN")
+
+var natsurl string = os.Getenv("NATS_ADDRESS")
+
 func main() {
 
 	// Connect to NATS
-	nc, err := nats.Connect("nats", nats.Name("K8s Metrics"), nats.Token("UfmrJOYwYCCsgQvxvcfJ3BdI6c8WBbnD"))
+
+	//, nats.Token("UfmrJOYwYCCsgQvxvcfJ3BdI6c8WBbnD")
```
Comment (avinashkna4) [inline client/k8smetrics_client.go:26]: remove this token  comment as well

File: clickhouse/clickhouse_client.go (hunk 1 above)
Comment (avinashkna4) [reply clickhouse/clickhouse_client.go:21]: also please test all the changes before merging it

**Review by avinashkna4 — CHANGES_REQUESTED** [review]

**Comment by vijeyash1** [comment]
Thank you for the feedback bro. Sure Will make the changes bro.

**Review by avinashkna4 — APPROVED** [review]

**Comment by jebinjeb** [comment]
rebuild

**Pull request merged**